extensions/amp-autocomplete/validator-amp-autocomplete.protoascii

```diff
@@ -106,3 +106,14 @@ tags: {  # amp-autocomplete JSON
     }
   }
 }
+tags: {  # amp-autocomplete disabled
```

Comment:
Could you add a comment to https://github.com/ampproject/amphtml/blob/master/validator/validator-main.protoascii#L1759 that "DIV" is also defined in this file.

Reply:
Resolving this and the other comment as this change has been removed :)